core/frontend/src/GeoServices.ts

```diff
@@ -266,6 +272,19 @@ export class GeoConverter {
     });
   }
 
+  /** Convert the specified geographic coordinates into iModel coordinates. */
+  public async geoCoordsToIModelCoords(geoPoints: XYZProps[]): Promise<PointWithStatus[]> {
```

Comment:
`convertToIModelCoords` ?

Reply:
I would second a change of name if possible since the converter can convert coordinates that are not latitude/longitude now if the user of the GeoConverter sets it otherwise.

Reply:
If you want me to rename something, please explain why.
As you know, I hate arguing about names.
I considered how to name this and my inclination was that this name makes it clear what both the source and target of the conversion are.

Reply:
@AlainRobertAtBentley what would you suggest as a replacement for the term "geo coords" then?

Reply:
@AlainRobertAtBentley please suggest changes to terminology of "geographic (lat/long) coordinate", arguments called `geoPoints`, etc.

Reply:
Initially the class and methods were intended to convert latitude/longitde coordinates to or from the iModel Coordinates. The class was modified so that it allows performing conversion from any Geographic Coordinate Reference System (GCRS) which may be latitude/longitude based or projected (cartesian). The term geoPoint hints to the fact it is latitude/longitude which may not be the case. Simply naming inPoint, outPoint or sourcePoint, targetPoint would be sufficient.
As for the GeoConverter and GeoServices names I do not mind as much their current name. Technically the GeoConverter could be named GeographicCoordinateReferenceSystemConverter but it would be overly verbose. Same reflection holds for GeoServices.